hermes-common/src/main/java/pl/allegro/tech/hermes/common/kafka/KafkaConsumerPoolConfig.java

```diff
@@ -12,20 +12,122 @@ public class KafkaConsumerPoolConfig {
     private final String securityMechanism;
     private final String securityProtocol;
     private final String saslJaasConfig;
+    private final boolean isSslEnabled;
+    private final String sslTrustStoreLocation;
+    private final String sslTrustStorePassword;
+    private final String sslKeyStoreLocation;
+    private final String sslKeyStorePassword;
+    private final String sslKeyPassword;
+    private final String sslProtocolVersion;
+    private final String sslEndpointIdentificationAlgorithm;
+
+    public String getSslKeyStoreCertificateChain() {
```

Comment:
Let's keep the order of new fields and getters/setters consistent with the existing ones.